src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -224,9 +224,8 @@ void CommandRequestImpl::onTimeOut() {
                                             function_id(),
                                             correlation_id(),
                                             mobile_api::Result::GENERIC_ERROR);
-
+  (*response)[strings::msg_params][strings::info] = info;
   AddTimeOutComponentInfoToMessage(response);
```

Comment:
@pold500 Well I guess that means that the test hits 1 of the return false conditions in this method, have you checked what condition was it?